packages/react-components/src/organisms/CalendarList.tsx

```diff
@@ -54,67 +70,102 @@ const orderList = css({
     borderTop: `1px solid ${steel.rgb}`,
   },
 });
+
+const showMoreStyles = css({
+  display: 'flex',
+  justifyContent: 'center',
+  paddingTop: `${15 / perRem}em`,
+  paddingBottom: `${15 / perRem}em`,
+  borderTop: `1px solid ${steel.rgb}`,
+});
+
 interface CalendarListProps {
   calendars: ReadonlyArray<CalendarResponse>;
-  page: 'calendar' | 'event' | 'group';
+  title: string;
+  description?: string;
+  hideSupportText?: boolean;
 }
-const CalendarList: React.FC<CalendarListProps> = ({ calendars, page }) => (
-  <div css={containerStyles}>
-    <Card>
-      <div css={headerStyles}>
-        <Headline3>
-          Subscribe to{' '}
-          {page === 'group'
-            ? "this group's"
-            : page === 'event'
-            ? "this event's"
-            : 'Groups on'}{' '}
-          calendar
-        </Headline3>
-      </div>
-      {page === 'calendar' && (
+const CalendarList: FC<CalendarListProps> = ({
+  calendars,
+  title,
+  description,
+  hideSupportText = false,
+}) => {
+  const truncateFrom = 5;
+  const [showMore, setShowMore] = useState(false);
+  const displayShowMoreButton = calendars.length > truncateFrom;
+
+  return (
+    <div css={containerStyles}>
+      <Card padding={false}>
+        <div css={headerStyles}>
+          <Headline3 noMargin>{title}</Headline3>
+        </div>
+        {description && (
+          <div css={subheaderStyles}>
+            <Paragraph accent="lead" noMargin>
+              {description}
+            </Paragraph>
+          </div>
+        )}
+        {!!calendars.length && (
+          <ul
+            css={[
+              orderList,
+              !displayShowMoreButton && {
```

Comment:
This is going to make warnings in the test output because a boolean is not serialized strings